target/linux/ramips/image/mt7621.mk

```diff
@@ -187,6 +187,25 @@ define Device/asus_rt-n56u-b1
 endef
 TARGET_DEVICES += asus_rt-n56u-b1
 
+define Device/beeline_smartbox-turbo-plus
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 124416k
```

Comment:
The main purpose of IMAGE_SIZE is for check-size, and here it is used by rootfs0.bin. So it has to be adjusted to 120320k.

Reply:
Fixed.